samples/request_tracking_codes.py

```diff
@@ -0,0 +1,12 @@
+from ..correios.client import Correios
+from ..correios.models.user import User, Service
+
+
+def get_tracking_codes(service, quantity):
+    olist_user = User('Your Company\'s Name', 'Your Company\'s CNPJ')
+    client = Correios('Your Correio\'s username', 'Your correio\'s password')
```

Comment:
use double-quote for strings to remove single quote escaping...

and add LICENSE header here...